ios/Classes/SwiftNewrelicMobilePlugin.swift

```diff
@@ -41,6 +41,7 @@ public class SwiftNewrelicMobilePlugin: NSObject, FlutterPlugin {
                 NewRelic.disableFeatures(NRMAFeatureFlags.NRFeatureFlag_WebViewInstrumentation)
             }
             if(args?["interactionTracingEnabled"] as! Bool == false) {
```

Comment:
we may want to avoid force unwrapping where possible `as!` could be replaced with `as?` here.